src/js/actions/userActions.js

```diff
@@ -136,6 +136,7 @@ export const getRoles = () => (dispatch, getState) =>
 
 export const createRole = roleData => dispatch => {
   let permissions = roleData.groups.map(group => ({ action: 'CREATE_DEPLOYMENT', object: { type: 'DEVICE_GROUP', value: group } }));
+  permissions.push({ action: 'http', object: { type: 'any', value: `any` } });
```

Comment:
@mzedel I believe this is fundamentally wrong; the right way to have a user limited to a set of groups is to combine that permission with another role (e.g. the observer/read-only, the CI, or whatever else). This hard-coded permission makes it impossible to combine the limit-to-group with other RBAC roles that limit features.

Reply:
understood... then I think I'll have to signal adding the observer role in the UI, as the groups and user management options won't be usable on their own

Reply:
@mzedel we started discussing with @merlin-northern a few additional "default" roles which can be used to create useful users without the "permit all". Keep you posted.